labgrid/remote/client.py

```diff
@@ -1208,6 +1209,9 @@ async def print_reservations(self):
             print("Reservation '{}':".format(res.token))
             res.show(level=1)
 
+    async def print_version(self):
+        print("Labgrid version", lg_version)
```

Comment:
Just an idea: we could additionally print the exporter version here, right? AFAICS there is no "coordinator version" exposed, is it?

Reply:
Currently, the client doesn't directly access the exporter.

Reply:
Too bad, okay.